internal/rbac/gen.go

```diff
@@ -0,0 +1,5 @@
+package rbac
+
+//go:generate env GO111MODULE=on go run yamldata.go -pkg rbac -o constants.go -lang go
+//go:generate gofmt -s -w constants.go
+//go:generate env GO111MODULE=on go run yamldata.go -pkg rbac -o ../../client/web/src/rbac/constants.ts -lang ts
```

Comment:
Nit: Probably don't need the `-pkg` arg on this one:
```suggestion
//go:generate env GO111MODULE=on go run yamldata.go -o ../../client/web/src/rbac/constants.ts -lang ts
```